### PR TITLE
LPS-104035

### PR DESCRIPTION
**Pull request by erickmonteiroo**

### CODE REVIEW

**Comment by erickmonteiroo** [comment]
ci:test:sf

**Comment by erickmonteiroo** [comment]
ci:test:relevant

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/account_entry/add_domains.jsp [hunk 1]
```diff
@@ -20,28 +20,30 @@
...
-	</aui:field-wrapper>
+			<div class="form-text">
+				<liferay-ui:message key="for-multiple-domains,-separate-each-domain-by-a-comma" />
+			</div>
+		</aui:field-wrapper>
 
-	<aui:button-row>
-		<aui:button onClick='<%= liferayPortletResponse.getNamespace() + "addDomains();" %>' primary="<%= true %>" value="save" />
+		<aui:button-row>
+			<aui:button onClick='<%= liferayPortletResponse.getNamespace() + "addDomains();" %>' type="submit" value="save" />
```
Comment (drewbrokke) [inline modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/account_entry/add_domains.jsp:41]: @erickmonteiroo it's probably best to move the function binding from here to the `onSubmit` attribute of the `liferay-frontend:edit-form` tag. That way, we're guaranteed to run the function no matter _how_ the form is submitted.

**Review by drewbrokke — CHANGES_REQUESTED** [review]
@erickmonteiroo looks good, I just have the one suggested change. See my inline comment.

**Review by drewbrokke — APPROVED** [review]
@erickmonteiroo LGTM 👍

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**